LuaRules/Gadgets/weapon_projectile_radar_homing.lua

```diff
@@ -22,15 +22,13 @@ local UNIT = string.byte('u')
 
 local projectiles = {}
 
-local projectileHomingDistance = {
-	[WeaponDefNames["shipcarrier_disarm_rocket"].id] = 600^2,
-	[WeaponDefNames["turretaaheavy_advsam"].id] = 1800^2,
-	[WeaponDefNames["amphraid_torpedo"].id] = 200^2,
-}
+local projectileHomingDistance = {}
 
 for wdid = 1, #WeaponDefs do
 	local wd = WeaponDefs[wdid]
-	if (not projectileHomingDistance[wdid]) and wd.tracks and
+	if (wd.customParams.radar_homing_distance) then
+		projectileHomingDistance[wdid] = tonumber(wd.customParams.radar_homing_distance)
```

Comment:
IMO do
```suggestion
		projectileHomingDistance[wdid] = tonumber(wd.customParams.radar_homing_distance)^2
```
and remove the ^2 from defs.